backend/SlothOrganizer/SlothOrganizer.Services/Utility/DateTimeService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using SlothOrganizer.Services.Abstractions.Utility;
+
+namespace SlothOrganizer.Services.Utility
+{
+    public class DateTimeService : IDateTimeService
+    {
+        public DateTime Now() => DateTime.Now;
```

Comment:
I recommend avoiding using the `` DateTime `` type when you want to represent a point in time. It may lead to issues when working with different time zones. The default type for time would be `` DateTimeOffset ``

More info here https://learn.microsoft.com/en-us/dotnet/standard/datetime/choosing-between-datetime

Reply:
Ok, I will replace DateTime with DateTimeOffset in a separate branch